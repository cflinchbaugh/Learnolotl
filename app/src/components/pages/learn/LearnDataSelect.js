import React from 'react';
import styled from 'styled-components';

import { Link } from 'react-router-dom';
import Button from 'buttons/Button';

const StyleWrapper = styled.div`

`

function LearnDataSelect (props) {
    let demoButtonData = {
            label: 'Japanese Demo',
            onClickFunction: props.handleDemoDataClick
        }
    return (
        <StyleWrapper className="learn-data-selectwrapper">
            <Button {...demoButtonData} />
        </StyleWrapper>
    );
}


export default LearnDataSelect;