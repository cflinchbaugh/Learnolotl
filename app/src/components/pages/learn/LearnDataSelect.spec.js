import React from 'react';
import {shallow} from 'enzyme';
import LearnDataSelect from './LearnDataSelect';
import ShallowRenderer from 'react-test-renderer/shallow';

describe('LearnDataSelect', () => {
    it('renders and matches snapshot', () => {
        const renderer = new ShallowRenderer();

        const learnDataSelectData = {
        }

        const tree = renderer.render(
            <LearnDataSelect {...learnDataSelectData}/>
        );

        expect(tree).toMatchSnapshot();
    });

    
});