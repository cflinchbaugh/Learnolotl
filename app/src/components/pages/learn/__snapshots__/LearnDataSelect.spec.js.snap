// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LearnDataSelect renders and matches snapshot 1`] = `
<styled.div
  className="learn-data-selectwrapper"
>
  <Button
    label="Japanese Demo"
    onClickFunction={undefined}
  />
</styled.div>
`;
